worker/src/main/scala/com/lucidchart/piezo/TriggerMonitoringModel.scala

```diff
@@ -39,23 +40,29 @@ class TriggerMonitoringModel(props: Properties) {
   def setTriggerMonitoringRecord(
     triggerKey: TriggerKey,
     triggerMonitoringPriority: TriggerMonitoringPriority,
-    maxSecondsInError: Int
+    maxSecondsInError: Int,
+    monitoringTeam: Option[String]
   ): Int = {
     val connection = connectionProvider.getConnection
     try {
       val prepared = connection.prepareStatement("""
         INSERT INTO trigger_monitoring_priority
-          (trigger_name, trigger_group, priority, max_error_time)
+          (trigger_name, trigger_group, priority, max_error_time, monitoring_team)
         VALUES
-          (?, ?, ?, ?)
+          (?, ?, ?, ?, ?)
         ON DUPLICATE KEY UPDATE
           priority = values(priority),
-          max_error_time = values(max_error_time)
+          max_error_time = values(max_error_time),
+          monitoring_team = values(monitoring_team)
       """)
       prepared.setString(1, triggerKey.getName)
       prepared.setString(2, triggerKey.getGroup)
       prepared.setInt(3, triggerMonitoringPriority.id)
       prepared.setInt(4, maxSecondsInError)
+      monitoringTeam match {
```

Comment:
nit: 

monitoringTeam.fold(prepared.setNull(5, java.sql.Types.VARCHAR))(team => prepared.setString(5, team))

Reply:
I generally tend to agree with [this article](https://lucidatlassian.atlassian.net/wiki/spaces/~jhenline/pages/52616397881/Prefer+not+to+use+Option.fold) by jhenline that argues that Option#fold is inconsistent and other ways of writing it are more readable